### ISSUE TITLE
Understatement about how running_var is computed in BatchNorm

### ISSUE THREAD

**Issue by vovaf709** [pytorch/pytorch#62399]
## 📚 Documentation

In the current documentation of `BatchNorm` the phrase "The standard-deviation is calculated via the biased estimator, equivalent to torch.var(input, unbiased=False)" is a little confusing. The thing is that according to the [source code](https://github.com/pytorch/pytorch/blob/420b37f3c67950ed93cd8aa7a12e673fcfc5567b/aten/src/ATen/native/Normalization.cpp#L223-L224) of `BatchNorm2d` (and other batchnorms) this layer uses biased variance estimator during training, but accumulates unbiased version in `running_var`. This can lead to misunderstanding for developers who implement their own layers and compare their realizations with the original.

**Comment by vovaf709** [comment]
If you think this flaw is serious enough, I could fix it
